.env.example

```diff
@@ -43,8 +43,13 @@ USERS_LOGIN_QUEUE_NAME=ulqueuename
 TEST_LOGIN_FISCAL_CODES=AAABBB01C02D345Z,EEEFFF01C02H345Z
 TEST_LOGIN_PASSWORD=secret
 FF_BONUS_ENABLED=1
+FF_CGN_ENABLED=1
```

Comment:
Could you add a small Feature Flag section inside the README that explains how it works?